src/bindings/python/tests/test_utils/test_utils.py

```diff
@@ -79,3 +80,9 @@ def deprecated_function4():
         deprecated_function3()
     with pytest.warns(DeprecationWarning, match="deprecated_function4 is deprecated and will be removed in version 2025.4. Use another function instead"):
         deprecated_function4()
+
+
+def create_filename_for_test(test_name):
+    python_version = str(sys.version_info.major) + "." + str(sys.version_info.minor)
+    filename = "./" + test_name[5:] + "_" + python_version
```

Comment:
```suggestion
    filename = "./" + test_name.replace("test", "") + "_" + python_version
```
Might be a littler cleaner solution, just an improvement.

Reply:
done